apps/typegpu-docs/src/components/ExampleView.tsx

```diff
@@ -126,6 +126,8 @@ export function ExampleView({ example, isPlayground = false }: Props) {
               codeEditorShowing ? 'max-h-[calc(50vh-3rem)]' : '',
             )}
           >
+            {def.elements.length === 0 ? <Canvas aspectRatio={1} /> : null}
```

Comment:
Could you add a comment here to signal that this is a temporary solution and why? For the future generations (us most likely) 🧒

```suggestion
            {/* Note: This is a temporary measure to allow for simple examples that do not require the @typegpu/example-toolkit package. */}
            {def.elements.length === 0 ? <Canvas aspectRatio={1} /> : null}
```